site/content/api/Tenant-API.md

```diff
@@ -0,0 +1,264 @@
+
+![Remove Tenant message flow](../tenant_RemoveTenantSuccess.png)
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on a *remove tenant* message in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name        | Mandatory | Location                 | Type     | Description |
+| :---------- | :-------: | :----------------------- | :------- | :---------- |
+| *subject*   | yes       | *properties*             | *string* | MUST be set to `remove`. |
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+
+**Response Message Format**
+
+A response to a *remove tenant* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response may contain the following status codes:
+
+| Code  | Description |
+| :---- | :---------- |
+| *204* | No Content, the tenant has been removed successfully. |
+| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+# Standard Message Properties
+
+Due to the nature of the request/response message pattern of the operations of the Tenant API, there are some standard properties shared by all of the request and response messages exchanged as part of the operations.
+
+## Standard Request Properties
+
+The following table provides an overview of the properties shared by all request messages regardless of the particular operation being invoked.
+
+| Name             | Mandatory | Location                 | Type         | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *subject*        | yes       | *properties*             | *string*     | MUST be set to the value defined by the particular operation being invoked. |
+| *correlation-id* | no        | *properties*             | *message-id* | MAY contain an ID used to correlate a response message to the original request. If set, it is used as the *correlation-id* property in the response, otherwise the value of the *message-id* property is used. |
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
+| *tenant-id*      | yes       | *application-properties* | *string*     | MUST contain the ID of the tenant that is subject to the operation. |
```

Comment:
should be renamed to *tenant_id* `
JMS based clients cannot set properties containing a `-` (dash) character in the property name ...

Reply:
adapted